plugins/sumneko_plugin.lua

```diff
@@ -0,0 +1,53 @@
+--@Author Rath
+--Made for Liquipedia
+--For use with sumneko-lua vscode/neovim extension
```

Comment:
Explain that this file is read automatically.

Reply:
Updated comment